infra_tooling_account/infra_tooling_account/infra_tooling_alerting_stack.py

```diff
@@ -77,22 +81,17 @@ def create_event_bus(self):
         )
 
         # EventBridge bus resource policy
-        # TODO: reuse existing settings reader
+        # TODO: use settings validation
```

Comment:
another approach would be to read and validate all settings just once - in cdk app.py before stacks.
After that we consider all settings as valid.

Not for this PR specifically - but for future, when we finalize settings classes structure